project/justfix_environment.py

```diff
@@ -396,6 +396,10 @@ class JustfixEnvironment(typed_environ.BaseEnvironment):
     # The origin of the NYC GeoSearch API.
     NYC_GEOSEARCH_ORIGIN: str = "https://geosearch.planninglabs.nyc"
 
+    CONTENTFUL_SPACE_ID: str = ""
+
+    CONTENTFUL_ACCESS_TOKEN: str = ""
```

Comment:
We should consider just using our actual space ID and access token as defaults here; because the access token is read-only and these credentials are intended for use in browser-side JS, they're effectively public (not secrets), and as such, just hard-coding them as defaults won't be a security hazard, and will also make it easier for developers to get up and running quickly.

Reply:
So this is kind of frustrating, I ended up adding the defaults to the WoW version of this PR, in https://github.com/JustFixNYC/who-owns-what/pull/482, and immediately got this email:

> ![image](https://user-images.githubusercontent.com/124687/122392251-b582bc80-cf41-11eb-9a54-964abb735a9a.png)

ARGH.  While I am pretty sure this isn't _actually_ a security hole, I really, really wish it were possible to just tell Contentful to make a space publicly readable so we didn't have to go through this false alarm stuff.